cpp/ps0000/ps0000_solv.cpp

```diff
@@ -1,10 +1,11 @@
-@iinclude <"iostream.h">   // incorrect
+// @iinclude <"iostream.h">   // incorrect
 
 #include <iostream>       // correct
 
 /* First error  : To call any library, you must use #include not @iinclude .
    Second error : The library used is placed directly inside these brackets < > Without quotation marks " " .
    Third error  : The correct library is this iostream not iostream.h .
+   2/2

```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`